tests/integration/test.sync_events.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
```

Comment:
This is a new test file, so it would need to be added to `tests/integration/index.html` in order to actually be tested in the browser. As is, it is only tested in node, because we do `mocha tests/integration/test*js`